src/components/Modal/index.js

```diff
@@ -1,14 +1,47 @@
-import React from 'react';
+import React, {PureComponent} from 'react';
```

Comment:
We weren't using `memo` before, so why start using `PureComponent` now?

Reply:
Yeah, it can be removed. `baseModal` is already pure.